src/actions/attacks/slam.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from src.common.point import Point
+from src.actions.action import Action
+from src.actions.deal_dmg import DealDmg
+from src.objects.object import Object
+from src.objects.npc import NPC
+from src.objects.agent import Agent
+
+SLAM_DMG = 10
+
+
+class Slam(Action):
+    """
+    Simple action for attacking.
+    """
+    object: Object
+
+    def __init__(self, object: Object, points: List[Point]) -> None:
+        super().__init__()
+        self.properties.object_id = object.properties.id
+        self.object = object
+        self.properties.points = points
```

Comment:
This thing is a little tricky. Right now front-end does not support deserialization of lists but I think it should not be a problem to add something like that. I guess it will be enough to have some helper function in `Deserializer` to call `strtok`.